### PR TITLE
tgui-next Shuttle Manipulator

### PR DESCRIPTION
**Pull request by actioninja**
## About The Pull Request

Replaces the shuttle manipulator ui with a tgui-next version.
This is a real beefy ui so I'm pring this separately for easier review.
https://cdn.discordapp.com/attachments/326831214667235328/635712680509964338/2019-10-20_22-34-24.mp4

## Why It's Good For The Game

Old one was *extremely* slow. This one is not.

## Changelog
:cl:
add: Replaced the shuttle manipulator ui with a tgui-next version
/:cl:

### CODE REVIEW

**Review by stylemistake — CHANGES_REQUESTED** [review]
Looks ok. Tested locally, compiles only on current master (due to Yarn and Node 12 LTS), so it needs rebasing.

File: tgui-next/packages/tgui/interfaces/ShuttleManipulator.js [hunk 1]
```diff
@@ -20,39 +20,39 @@ export const ShuttleManipulator = props => {
           <Section>
             <table>
               {shuttles.map(shuttle => (
-                <tr key={shuttle.id + '_row'}>
-                  <td key={shuttle.id + '_jmprow'}>
+                <tr key={shuttle.id}>
+                  <td key={shuttle.id}>
```
Comment (stylemistake) [inline tgui-next/packages/tgui/interfaces/ShuttleManipulator.js]: Oy, I have just noticed you assign a key to all of the elements. Assign keys only to top level array elements, i.e. `tr`, rest don't matter. Linter will help you.

Comment (actioninja) [reply tgui-next/packages/tgui/interfaces/ShuttleManipulator.js]: make sense

File: tgui-next/packages/tgui/interfaces/ShuttleManipulator.js [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+                              {!!actualTemplate.admin_notes && (
+                                <LabeledList.Item label="Admin Notes">
+                                  {actualTemplate.admin_notes}
+                                </LabeledList.Item>
+                              )}
+                            </LabeledList>
+                          )}
+
+                        </Section>
+                      ); })}
```
Comment (stylemistake) [inline tgui-next/packages/tgui/interfaces/ShuttleManipulator.js]: move those closing brackets to the next line

File: tgui-next/packages/tgui/interfaces/ShuttleManipulator.js [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+                  </Tabs.Tab>
+                ); })(templateObject)}
```
Comment (stylemistake) [inline tgui-next/packages/tgui/interfaces/ShuttleManipulator.js]: same for this one, looks pretty ugly and hard to follow

**Pull request merged**